forge-gui/src/main/java/forge/model/FModel.java

```diff
@@ -404,50 +404,11 @@ public static ItemPool<PaperCard> getContraptionPool() {
      */
     public static void loadDynamicGamedata() {
         if (!CardType.Constant.LOADED.isSet()) {
+            
             final List<String> typeListFile = FileUtil.readFile(ForgeConstants.TYPE_LIST_FILE);
```

Comment:
Always try to avoid duplicating code, especially if it contains non-trivial logic as it's much harder to maintain consistently if variants exist in multiple places:
in this case I don't see why you can't just apply `FileSection.parseSections` here too, so the main file and edition extensions don't need to be treated differently...